### PR TITLE
fix openh264 folder rename issue

### PR DESCRIPTION
**Pull request by huili2**
att. 
Change the test code to re-enable the fuzzy test for openh264.

### CODE REVIEW

**Comment by DavidKorczynski** [comment]
The CI is complaining about a missing header file in https://github.com/google/oss-fuzz/blob/master/projects/openh264/decoder_fuzzer.cpp -- would it perhaps be possible to move the fuzzer upstream?

**Comment by huili2** [comment]
anybody could have a look if my second commit fixed the issue? Thanks.

File: projects/openh264/decoder_fuzzer.cpp [hunk 1]
```diff
@@ -1,3 +1,20 @@
+/*
+# Copyright 2021 Google LLC
```
Comment (DavidKorczynski) [inline projects/openh264/decoder_fuzzer.cpp:2]: Many apologies for the delay here! Could you do a minor nit and change this to 2022? After that we can merge this.

**Review by DavidKorczynski — CHANGES_REQUESTED** [review]

File: projects/openh264/decoder_fuzzer.cpp (hunk 1 above)
Comment (DavidKorczynski) [reply projects/openh264/decoder_fuzzer.cpp:2]: ah sorry - you moved the file instead of adding  -- I'll merge this in.

**Review by DavidKorczynski — APPROVED** [review]

**Pull request merged**